packages/webview_flutter/webview_flutter_wkwebview/example/lib/web_view.dart

```diff
@@ -156,6 +157,32 @@ class WebView extends StatefulWidget {
   /// By default `allowsInlineMediaPlayback` is false.
   final bool allowsInlineMediaPlayback;
 
+  /// Controls whether navigation should be limited to App-Bound Domains on iOS.
+  ///
+  /// This field is ignored on Android as this is a iOS specific feature
+  ///
+  /// By default `limitsNavigationsToAppBoundDomains` is false.
+  ///
+  /// Caveats:
+  ///
+  ///   * Requires app-bound domains to be added to iOS Info.plist
+  ///     * Example:
+  ///         ```xml
+  ///           <plist version="1.0">
+  ///           <dict>
+  ///             <key>WKAppBoundDomains</key>
+  ///             <array>
+  ///                 <string>example1.com</string>
+  ///                 <string>example2.org</string>
+  ///                 ...
+  ///             </array>
+  ///           </dict>
+  ///         ```
+  ///   * `Array` of App-Bound Domains is limited to 10 domains
+  ///   * Content supplied by the app through local files, data URLs and HTML String are always treated
+  ///     as App-Bound Domains, and do not need to be listed.
```

Comment:
You don't need to re-document the feature in the example. If people want to know what the feature does, they should look at the API documentation.